bin/process.py

```diff
@@ -59,7 +59,7 @@ class Organizer(FrontmatterModel):
     title: Optional[str]
     twitter: Optional[str]
     website: Optional[str]
-
+    mastodon: Optional[str]
```

Comment:
```suggestion
    mastodon: Optional[str]

```

Leaving that blank line in there for black formatting purposes